packages/typera-express/src/parser.ts

```diff
@@ -21,6 +21,13 @@ function getHeaders(e: RequestBase): any {
   return new Proxy(e.req, {
     get: (target, field) =>
       typeof field === 'string' ? target.get(field) : undefined,
+    getOwnPropertyDescriptor() {
+      return {
+        enumerable: true,
+        configurable: true,
+      }
+    },
+    ownKeys: (target) => Object.keys(target.headers),
```

Comment:
These are probably not needed anymore, because the exact codecs are disallowed.

Reply:
Yep, this last commit is not needed to fix the codec issue per se. It just makes the Proxy a bit more well-behaved. I would be surprised if I were to call `Object.keys` on something with a type of eg.`{ accept: string, cookie: string }` and get back `["httpVersion", "res", "next", "aborted", "statusCode"]` etc.